dist/ember-charts.js

```diff
@@ -12,31 +12,56 @@ var _ref;
-  var buffer = '', stack1, hashContexts, hashTypes, options, helperMissing=helpers.helperMissing, escapeExpression=this.escapeExpression;
-
-
-  data.buffer.push("<svg ");
-  hashContexts = {'width': depth0,'height': depth0};
-  hashTypes = {'width': "STRING",'height': "STRING"};
-  options = {hash:{
-    'width': ("outerWidth"),
-    'height': ("outerHeight")
-  },contexts:[],types:[],hashContexts:hashContexts,hashTypes:hashTypes,data:data};
-  data.buffer.push(escapeExpression(((stack1 = helpers['bind-attr'] || depth0['bind-attr']),stack1 ? stack1.call(depth0, options) : helperMissing.call(depth0, "bind-attr", options))));
-  data.buffer.push(">\n  <g class=\"chart-viewport\" ");
-  hashContexts = {'transform': depth0};
-  hashTypes = {'transform': "STRING"};
-  options = {hash:{
-    'transform': ("transformViewport")
-  },contexts:[],types:[],hashContexts:hashContexts,hashTypes:hashTypes,data:data};
-  data.buffer.push(escapeExpression(((stack1 = helpers['bind-attr'] || depth0['bind-attr']),stack1 ? stack1.call(depth0, options) : helperMissing.call(depth0, "bind-attr", options))));
-  data.buffer.push("></g>\n</svg>\n");
-  return buffer;
-  
-});
+Ember.TEMPLATES["chart"] = Ember.HTMLBars.template((function() {
+  return {
+    isHTMLBars: true,
+    blockParams: 0,
+    cachedFragment: null,
+    hasRendered: false,
+    build: function build(dom) {
+      var el0 = dom.createDocumentFragment();
+      dom.setNamespace("http://www.w3.org/2000/svg");
+      var el1 = dom.createElement("svg");
+      var el2 = dom.createTextNode("\n  ");
+      dom.appendChild(el1, el2);
+      var el2 = dom.createElement("g");
+      dom.setAttribute(el2,"class","chart-viewport");
+      dom.appendChild(el1, el2);
+      var el2 = dom.createTextNode("\n");
+      dom.appendChild(el1, el2);
+      dom.appendChild(el0, el1);
+      var el1 = dom.createTextNode("\n");
+      dom.appendChild(el0, el1);
+      return el0;
+    },
+    render: function render(context, env, contextualElement) {
+      var dom = env.dom;
+      var hooks = env.hooks, element = hooks.element;
+      dom.detectNamespace(contextualElement);
+      var fragment;
+      if (env.useFragmentCache && dom.canClone) {
+        if (this.cachedFragment === null) {
+          fragment = this.build(dom);
+          if (this.hasRendered) {
+            this.cachedFragment = fragment;
+          } else {
+            this.hasRendered = true;
+          }
+        }
+        if (this.cachedFragment) {
+          fragment = dom.cloneNode(this.cachedFragment, true);
+        }
+      } else {
+        fragment = this.build(dom);
+      }
+      var element0 = dom.childAt(fragment, [0]);
+      var element1 = dom.childAt(element0, [1]);
+      element(env, element0, context, "bind-attr", [], {"width": "outerWidth", "height": "outerHeight"});
+      element(env, element1, context, "bind-attr", [], {"transform": "transformViewport"});
+      return fragment;
+    }
+  };
+}()));
```

Comment:
This code is compiled, so changing it directly won't work. If you run `grunt dist` immediately before committing your changes you'll see that this code gets reset by it.